Sources/PublicInterface/Session.swift

```diff
@@ -54,9 +56,9 @@ public struct Session: Codable {
         public let accounts: [String]
         public let chainId: Int
         public let peerId: String
-        public let peerMeta: ClientMeta
+        public let peerMeta: ClientMeta?
```

Comment:
This one is related to https://docs.walletconnect.org/tech-spec#session-request
According to specs:
_The wc_sessionRequest will expect a response with the following parameters:_
```
interface WCSessionRequestResponse {
  id: number;
  jsonrpc: "2.0";
  result: {
    peerId: string;
    peerMeta: ClientMeta;
    approved: boolean;
    chainId: number;
    accounts: string[];
  };
}

```
@pedrouid  is it required by protocol to always have peerMeta in the response? If not what other parameters can be optional?